addons/website/models/website.py

```diff
@@ -1341,3 +1341,80 @@ def _get_cached_values(self):
 
     def _get_cached(self, field):
         return self._get_cached_values()[field]
+
+    def _get_html_fields(self):
+        html_fields = [('ir_ui_view', 'arch_db')]
+        cr = self.env.cr
+        cr.execute(r"""
+            SELECT f.model, f.name
+                    FROM ir_model_fields f
+                    JOIN ir_model m ON m.id = f.model_id
+                    WHERE f.ttype = 'html'
+                    AND f.store = true
+                    AND m.transient = false
+                    AND f.model NOT LIKE 'ir.actions%'
+                    AND f.model != 'mail.message'
+        """)
```

Comment:
weird indent ;-)
```suggestion
            SELECT f.model, f.name
              FROM ir_model_fields f
              JOIN ir_model m ON m.id = f.model_id
             WHERE f.ttype = 'html'
               AND f.store = true
               AND m.transient = false
               AND f.model NOT LIKE 'ir.actions%'
               AND f.model != 'mail.message'
        """)
```
Or anything else you want but right now it seems weird